keras_nlp/models/bert/bert_models.py

```diff
@@ -259,215 +215,83 @@ def get_config(self):
                 "max_sequence_length": self.max_sequence_length,
                 "num_segments": self.num_segments,
                 "dropout": self.dropout,
-                "cls_token_index": self.cls_token_index,
```

Comment:
This is a bug for sure, but one we fix on https://github.com/keras-team/keras-nlp/pull/385, to keep the PRs separate.